packages/components/src/Tabs/Tab.tsx

```diff
@@ -19,25 +19,22 @@ export const Tab = ({ item, state }: TabProps) => {
   const { tabProps, isSelected } = useTab({ key }, state, ref);
   const disabled = tabProps['aria-disabled'];
   const { hoverProps, isHovered } = useHover({
-    isDisabled: disabled as boolean,
+    isDisabled: (disabled as boolean) || isSelected,
```

Comment:
@OsamaAbdellateef why also `isSelected` ?

Reply:
i wanna disable the hover if the tab is `selected` 
i guess it shouldn't have `hover` effect when it is already `selected`